scripts/rollup/build.js

```diff
@@ -435,6 +437,9 @@ rimraf('build', async () => {
   try {
     // create a new build directory
     fs.mkdirSync('build');
+    // create the temp directory for local npm packing and unpacking
+    // in operating system's default temporary directory
+    fs.mkdirSync(npmPackagesTmpDir);
```

Comment:
What if it's not empty, and the previous build terminated abruptly before `rimraf` got a chance to run?

I think ideally we should create a new directory every time, with random uuids as folder names.

Reply:
What do you think about removing both `build` and `temp` before building, something like ``rimraf(`{build, ${npmPackagesTmpDir}}`, async () => {...})``
Passing the uuided path to all build functions feels quite cumbersome?

Reply:
It feels a bit wrong to me that if you have two checkouts of React, running `yarn build` in them at the same time might produce conflicts. So I'd rather pass it around.

Reply:
Thanks for pointing this out! Please review.